src/git.md

```diff
@@ -263,6 +264,67 @@ You also may want to squash just the last few commits together, possibly
 because they only represent "fixups" and not real changes. For example,
 `git rebase --interactive HEAD~2` will allow you to edit the two commits only.
 
+### `git range-diff`
+
+After completing a complicated rebase, or even a relatively simple one, you may
```

Comment:
```suggestion
After a rebase, you may
```

Reply:
Or maybe:

```suggestion
After completing a rebase, and before pushing up your changes,
```

Because `git range-diff master @{u} HEAD` only works before you push.

Reply:
Sounds good to me :)